packages/utils/telemetry-utils/src/logger.ts

```diff
@@ -533,14 +536,17 @@ function getValidTelemetryProps(obj: any): ITelemetryProperties {
  *
  * PLEASE take care to properly tag properties set on this object
  */
-export class LoggingError extends Error implements ILoggingError {
+export abstract class LoggingError extends Error implements ILoggingError, Omit<IErrorBase, "sequenceNumber"> {
     private readonly __isFluidLoggingError__ = 1;
 
+    public abstract readonly errorType: string;
```

Comment:
I'm not sure about errorType here (and abstract class).
Basically the main purpose of errorType, or rather where it is used - communicate to host about what error happens on Container.load() and Container's "closed" event paths. We lot same thing to telemetry, which is fine, but it's not the primary reason for its existence.

LoggingError on the other hand should likely be viewed as an error object that helps us track what is safe and not save to be logged to telemetry. In general, it does not have to conform to requirements above. And vice versa, the error we log to host does not have to have anything safe to be logged to telemetry. While it's true that likely 99% of usage would overlap here, I think I'd prefer logging layer (this layer) have zero knowledge of above. If needed, we can add another derived class on top of it somewhere else (container-utils?) that does rely on both concepts.

Reply:
Yes I got more clarity on the separate abstractions of errorType/interfaces v. LoggingError as I worked on the PR, and reading what you're saying here I agree on reverting this.

Reply:
Another hint is the policy check failure which is about a layer-check violation :)